tests/test_config.py

```
@@ -53,7 +53,7 @@ def no_optional_config() -> dict:
     return {
         "modules": ["intel-compiler/2021.1.1", "netcdf/4.7.4", "openmpi/4.1.0"],
         "realisations": [
-            {"repo": {"svn": {"branch_path": "trunk"}}},
+            {"repo": {"svn": {"branch_path": "trunk"}}, "model_output_name": True},
```

Ideally, this should return an error because just using the name "trunk" for the model output name should not be allowed.

---

I have added in the validation function for checking model output name derived from config file, along with tests.